docs/source/using-executorch-building-from-source.md

```diff
@@ -56,13 +56,21 @@ Or alternatively, [install conda on your machine](https://conda.io/projects/cond
    conda create -yn executorch python=3.10.0 && conda activate executorch
    ```
 
-## Install ExecuTorch pip package from Source
+## Install ExecuTorch pip package from source
    ```bash
    # Install ExecuTorch pip package and its dependencies, as well as
    # development tools like CMake.
    # If developing on a Mac, make sure to install the Xcode Command Line Tools first.
+   # Intel-based macOS systems require building PyTorch, Torchvision, and Torchaudio from source (see below)
    ./install_executorch.sh
    ```
+  
+  Use the [`--use-pt-pinned-commit` flag](https://github.com/pytorch/executorch/blob/main/install_executorch.sh) to install Executorch with an existing PyTorch build.  
+  See the [PyTorch instructions](https://github.com/pytorch/pytorch#installation) on how to build PyTorch from source.
+
+   ```bash
+   ./install_executorch.sh --use-pt-pinned-commit
+   ```
```

Comment:
Insert the Intel-based instruction under here... 

Something like: 

For Intel-based instructions, use `./install_executorch.sh --use-pt-pinned-commit --minimal`